gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,86 @@ Features marked with a ☁️ require a GitKraken Account, with access level bas
 
 ---
 
+<a id="v14-8"></a>
+
+## Version 14.8
+
+#### Thursday, February 8, 2024
+
+The 14.8 release of GitLens feature several enhancements to make easier to use with worktrees, views diffs and pull-requests, and enterprise improvements for Cloud Patches.
+
+<img src="/wp-content/uploads/gl-14-8-hero.png" class="img-responsive center img-bordered">
+
+### Worktree Workflow Improvements ✨
+
+One of the most notable improvements is to worktrees. GitLens 14.8 simplifies the process of creating and opening worktrees by reducing the number of steps and options required. There are also new options to easily create a worktree when switching branches, and copy current working changes to an existing worktree. These worktree enhancements will help streamline your workflow.
```

Comment:
```suggestion
This release streamlines and upgrades the experience of working with and between worktrees. GitLens 14.8 simplifies the process of creating and opening worktrees by reducing the number of steps and options required. There are also new options to easily create a worktree when switching branches, and to copy current working changes to an existing worktree.
```

Reply:
Now that I'm looking at my suggestion, I'm realizing the first two sentences of the paragraph are feeling a bit redundant. May need to shuffle statements around a bit.